cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -21,15 +21,17 @@ desiredState:
         dnsPolicy: "Default"  # Don't use cluster DNS.
         containers:
           - name: etcd
-            image: quay.io/coreos/etcd:v2.0.3
+            image: gcr.io/google_containers/etcd:2.0.9
```

Comment:
@thockin See https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/images/etcd/Dockerfile

This is the same etcd that we use for apiserver. I'm not sure what's different from upstream though.